### DISCUSSION TITLE
problem with apt-offline doing --upgrade-type dist-upgrade

### DISCUSSION THREAD

**Discussion by Watcha31** [rickysarraf/apt-offline#221] [Q&A]
Hi everyone,

On Ubuntu 18.04 I had no problem doing 
- apt-offline set [yyyymmdd]_dist-upgrade.sig --upgrade-type dist-upgrade

On ubuntu 22.04 I had to change apt-offline_1.8.5-1_all.deb to make update.zip OK, but when I am doing 
apt-offline set [yyyymmdd]_dist-upgrade.sig --upgrade-type dist-upgrade --verbose I have :

VERBOSE: Namespace(verbose=True, quiet=False, set='20230921_dist-upgrade.sig', set_simulate=False, set_install_packages=None, set_install_src_packages=None, src_build_dep=False, set_install_release=None, set_update=False, set_upgrade=False, upgrade_type='dist-upgrade', generate_changelog=False, apt_backend='apt-get', func=<function setter at 0x7f6602a2c310>)
ERROR: No option specified to set command

I tried with python3-magic_0.4.24-2_all.deb, but without success, what is missing for dist-upgrade works ?

**Comment by rickysarraf** [comment]
Yes. We changed the defaults to be inline with the GUI. But more than that, the reason to change the defaults was  to be more explicit by the user.

For more context, please see:

* https://github.com/rickysarraf/apt-offline/commit/2cab46a31b65c1d4831da3b0b0ae0d7a2e4004dd
* https://github.com/rickysarraf/apt-offline/issues/136

To resolve your issue, just call the same set of options along with and additional option, `--upgrade`

**Reply by Watcha31** [reply]
Thanks very much I have to write my new documentation on Ubuntu 22.04 with apt-offline.
Have a good day
